### ISSUE TITLE
websockets.py bug

### ISSUE THREAD

**Issue by WebDesZ** [spesmilo/electrum#1559]
Hello. I followed the tutorial on http://docs.electrum.org/en/latest/merchant.html
Websocket seems to run fine (netstat -> 0.0.0.0:9999)

But I get following Error message and it seems as electrum doesn't "answer" after the client sends it the id. (e.g. id:adc60774cd)

```
Exception in thread Thread-12:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/local/lib/python2.7/dist-packages/electrum/websockets.py", line 86, in reading_thread
    self.server.send_request(self, request)
AttributeError: 'NetworkServer' object has no attribute 'send_request'

message received id:adc60774cd
```

**Comment by ecdsa** [comment]
@kyuupichan I believe this is caused by chages you made recently, can you have a look?

**Comment by kyuupichan** [comment]
Nope, not me @ecdsa.  Seems to be your daemon refactoring.

**Comment by ecdsa** [comment]
indeed, you are right

**Comment by ecdsa** [comment]
this is fixed, see 2934fd17be660887e6d0154d10dfb31310e18532 2a29d1e5e70fecbeb59e75a943b06d3303c64ad4 and 1207ad3ba41ed25cbc2d91b12fb2b45a431357f0
